addons/mail/static/src/core/common/message_reaction_menu.js

```diff
@@ -32,17 +32,17 @@ export class MessageReactionMenu extends Component {
         useExternalListener(document, "keydown", this.onKeydown);
         onExternalClick("root", () => this.props.close());
         useEffect(
-            (reactions) => {
-                const activeReaction = reactions.find(
+            (reactionsLength) => {
```

Comment:
I would just rely on `this.props.message` in the scope, not the param of dependency.
While it's shorter, it's an additional variable, while using `this.props.message` makes it easier to see it comes from the same source, which is message record